### PR TITLE
APP-1332: Forever top bar

### PR DESCRIPTION
**Pull request by hugokallstrom**

### CODE REVIEW

File: app/src/androidTest/java/com/hedvig/app/feature/embark/previousinsurer/PreviousInsurerTest.kt [hunk 1]
```diff
@@ -28,11 +26,6 @@ class PreviousInsurerTest : TestCase() {
     @get:Rule
     val apolloCacheClearRule = ApolloCacheClearRule()
 
-    @get:Rule
```
Comment (hedvigoscar) [inline app/src/androidTest/java/com/hedvig/app/feature/embark/previousinsurer/PreviousInsurerTest.kt:31]: The changes to these tests seem unrelated to this PR 🤔

File: app/src/main/java/com/hedvig/app/feature/referrals/ui/tab/ReferralsAdapter.kt [hunk 2]
```diff
@@ -462,14 +471,35 @@ class ReferralsAdapter(
...
+                onBannerClicked: () -> Unit
             ) {
                 binding.retry.setHapticClickListener {
                     tracker.reload()
                     reload()
                 }
             }
         }
+
+        class InfoBanner(parent: ViewGroup) : ViewHolder(parent.inflate(R.layout.view_info_banner)) {
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/referrals/ui/tab/ReferralsAdapter.kt:484]: Should we implement this using Compose instead?

File: app/src/main/java/com/hedvig/app/feature/referrals/ui/tab/ReferralsAdapter.kt [hunk 3]
```diff
@@ -462,14 +471,35 @@ class ReferralsAdapter(
                 data: ReferralsModel,
                 reload: () -> Unit,
                 tracker: ReferralsTracker,
-                marketManager: MarketManager
+                marketManager: MarketManager,
+                onBannerClicked: () -> Unit
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/referrals/ui/tab/ReferralsAdapter.kt:475]: This could perhaps be added directly when creating the `ViewHolder` instead of upon each call to `bind()`.

**Review by hedvigoscar — COMMENTED** [review]
Looks good! Some questions, but nothing major.

File: app/src/main/java/com/hedvig/app/feature/referrals/ui/tab/ReferralsAdapter.kt (hunk 2 above)
Comment (hugokallstrom) [reply app/src/main/java/com/hedvig/app/feature/referrals/ui/tab/ReferralsAdapter.kt]: Yes :D
